xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.414.xml

```diff
@@ -107,12 +107,19 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
+         <ab>
+            <lb n="1"/><persName><name type="praenomen" key="Gaius"><expan><abbr>C</abbr><ex>aius</ex></expan></name> <name type="gentilicium" key="Matho">Matho</name> <expan><abbr>C</abbr><ex>aius</ex></expan> <expan><abbr>f</abbr><ex>ilius</ex></expan>
+            <lb n="2"/><expan><abbr>Ter</abbr><ex>etina tribu</ex></expan> <name type="cognomen" key="Clemens">Clemens</name></persName>
+            <lb n="3"/>Missicius ex <expan><abbr>coh</abbr><ex>orte</ex></expan> <num xml:lang="lat" value="2"><hi rend="supraline">II</hi></num> <expan><abbr>pr</abbr><ex>aetoria</ex></expan>.
```

Comment:
if `missicius` is a position/title, shouldn't it not be capitalized? (My feeling is that the capitalization would suggest it's a name.)